### DISCUSSION TITLE
官网文档很多页面无法打开

### DISCUSSION THREAD

**Discussion by Tan-Pchan** [hhyo/Archery#2626] [General]
![image](https://github.com/hhyo/Archery/assets/68987741/b9a8a70e-2b33-427c-a709-f0150f268d07)

**Comment by LeoQuote** [comment]
wiki 能正常打开吗? 是哪个页面引用了这个网址?

**Reply by Tan-Pchan** [reply]
wiki正常打开。想找关于系统配置项的说明。
入口：https://github.com/hhyo/archery/wiki/upgrade
![image](https://github.com/hhyo/Archery/assets/68987741/5e7f58c3-653d-4661-99a2-ecddf319c563)

**Comment by blight19** [comment]
官网差不多全部打不开了

**Reply by blight19** [reply]
https://archerydms.com/#_5

**Reply by Tan-Pchan** [reply]
刚试了下，可以正常打开了。
